Win32.xs

```diff
@@ -1682,6 +1685,228 @@ XS(w32_IsDeveloperModeEnabled)
     XSRETURN_NO;
 }
 
+#if !defined(__GNUC__) || (((100000 * __GNUC__) + (1000 * __GNUC_MINOR__)) >= 408000)
```

Comment:
Instead if repeating the same condition 3 times in this file, I would rather use one of the symbols defined by `winhttp.h` in the latter checks, e.g.

```suggestion
#ifdef WINHTTPAPI
```

Or one of the symbols you are already using:

```suggestion
#ifdef WINHTTP_AUTOPROXY_AUTO_DETECT
```

Reply:
> Instead if repeating the same condition 3 times in this file, I would rather use one of the symbols defined by `winhttp.h` in the latter checks, e.g.

Good suggestion.  Now done.  

The CI tests failed trying to install Strawberry Perl.  It seems to have been a simple download failure.